src/Pack/Runner/Uninstall.idr

```diff
@@ -0,0 +1,26 @@
+module Pack.Runner.Uninstall
+
+import Pack.Config.Types
+import Pack.Core.IO
+import Pack.Core.Logging
+import Pack.Core.Types
+
+%hide Pack.Config.Types.Env.packDir
+
+%default total
+
+--------------------------------------------------------------------------------
+--          Uninstalling Pack
+--------------------------------------------------------------------------------
+
+export covering
+uninstallPack :
+     {auto _ : HasIO io}
+  -> {auto _ : Env}
```

Comment:
This function does not require environment to be able to work, we should demand only really what we use:

```suggestion
  -> {auto _ : LogRef}
  -> {auto _ : PackDir}
```

Reply:
Great, this should be addressed via 1b99a87fb55a97b8ede6fbfb1a6e2d6d820e756d.